### DISCUSSION TITLE
More crawler husks

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#6304] [Ideas and suggestions]
Crawler husks only spawn in caves currently, and normal crawlers are barely an annoyance because of how easy it is to get rid of them and recover from any encounter with them in the late game. Maybe adding some crawlerhusk swarms on later difficulties is a good idea?
Adding some crawlerhusk spawns to wrecks wouldn't do much harm either, even in the early game.

(Could also increase the overall damage the crawlerhusk does to make it equal or higher than the normal crawlers, to make it just a direct upgrade in difficulty)

**Comment by ghost** [comment]
I personally dislike crawler husks to begin with, the thing with normal husks is you can keep your distance to avoid being infected. 

crawler husks are too fast for that.

**Reply by ViktorShield** [reply]
What about just using a ship weaponry, fleeting, using a riot shotgun? Maybe consider using an antidote?
Pff, let's just melee crawlers until one of us falls dead

**Comment by LePiromano** [comment]
This suggestion became true, now crawler husks have a chance to appear instead of common crawlers.

**Comment by Hazmat616** [comment]
Husk symbiosis goes brr
